.github/workflows/docs.yml.jinja

```diff
@@ -7,67 +9,36 @@ on:
       publish:
         default: false
         type: boolean
-      version:
```

Comment:
Do we still need a way to set the version manually? Suppose we want to push a simple fix to the docs or the docs build failed but the rest of the release went through and published packages. Then we might want to manually run the workflow and upload the docs and show a specific version on the page.

Reply:
Yes, this should *not* be removed!

Reply:
How do you suggest we implement this in practise? The sphinx theme uses the version of the package it imports. Would you then patch the version of the e.g. plopp package in the CI workflow?

Reply:
Not sure I understand your question.

Reply:
Question is if I set a version number in a parameter box in the gh action, I need to propagate it to the sphinx for building. Currently sphinx just automatically uses the `plopp.__version__` (I think?) of the package (it's a `{{ version }}` template variable in the `_templates/doc_version.html` file.

Should I change the `doc_version.html` file to use something like an environment variable instead of the `{{ version }}`? (if that's even possible?)

Reply:
I don't think you need to change anything? The version set here is what is `pip install`'ed, so Sphinx will automatically get that version?

Reply:
So are you saying that I should have something like if ther version is set, then `tox --skip-pkg-install -e docs -- {{projectname}}==${VERSION}` and if not, just so `tox -e docs`?

Reply:
Yes. I don't think you need to change anything in the existing workflow, just change the condition from checking "publish" to check if the version is set? Or maybe the branch?

Reply:
So do we also need to keep the branch option as well?

Reply:
Yes, why would you want to change it?

Reply:
A simple 'Yes' would have been enough.

But to answer the second part: Because it was removed in plopp, and none of the comments above were talking about the `branch` option, so I thought it was fine. Now I see that I need to revert some changes in Plopp as well.